client/utils/account-fees.tsx

```diff
@@ -281,7 +281,7 @@ export const formatAccountFeesDescription = (
 		} );
 	}
 
-	return feeDescription;
+	return sprintf( __( 'From %s', 'woocommerce-payments' ), feeDescription );
```

Comment:
Since we're wrapping `feeDescription` in `sprintf` above, can we try to do it in a single place and keep this line as plain return only?

Reply:
Fixed. Added 'From' prefix when function is called, so we don't have to hardcode it. This way, the custom formatting won't interfere with 'From' prefix.